.github/workflows/publish_pypi.yml

```diff
@@ -16,9 +16,20 @@ on:
         required: false
         default: false
 
+# Set options available for all jobs that use cibuildwheel
+env:
+  # Increase pip debugging output, equivalent to `pip -vv`
+  CIBW_BUILD_VERBOSITY: 2
+  # Disable build isolation to allow pre-installing build-time dependencies.
+  # Note: CIBW_BEFORE_BUILD must be present in all jobs using cibuildwheel.
+  CIBW_BUILD_FRONTEND: "pip; args: --no-build-isolation"
```

Comment:
The reason for disabling build isolation is that we have an unusual Pythonic dependency that is being used to link to the `idaklu` target, and therefore it needs to be found in a standard location. Doing this via `cibuildwheel` is fine because it creates a new virtual environment per Python version when building the wheels which negates interferences with the build-time dependencies: they are pre-installed into said environment with the `CIBW_BEFORE_BUILD` command to counter the lack of build-time isolation.